### PR TITLE
Changed content four column based on new requirements

### PR DESCRIPTION
**Pull request by kirdia**

### CODE REVIEW

File: classes/controller/blocks/class-p4bks-blocks-contentfourcolumn-controller.php [hunk 1]
```diff
@@ -150,33 +62,41 @@ public function prepare_fields() {
...
+			if ( $query->have_posts() ) {
+
+				$posts = $query->get_posts();
+
+				foreach ( $posts as $post ) {
+
+					$post->alt_text  = '';
+					$post->thumbnail = '';
+
+					if ( has_post_thumbnail( $post ) ) {
```
Comment (sagarsdeshmukh) [inline classes/controller/blocks/class-p4bks-blocks-contentfourcolumn-controller.php]: Better if we used post ID here. So if `( has_post_thumbnail( POST_ID_HERE ) ) {` @koyan - Do you think same issue like article thumbnails can occur if we use this ?

Comment (koyan) [reply classes/controller/blocks/class-p4bks-blocks-contentfourcolumn-controller.php]: Have you tested it and it doesn't work as it should? 
From the documentation : https://developer.wordpress.org/reference/functions/has_post_thumbnail/  I see that this function takes either the ID or the post object itself.

Comment (sagarsdeshmukh) [reply classes/controller/blocks/class-p4bks-blocks-contentfourcolumn-controller.php]: Yes both will work. My question is which is more better :)

Comment (koyan) [reply classes/controller/blocks/class-p4bks-blocks-contentfourcolumn-controller.php]: If both work, then using the post is better. If you look at the code of the function, if you pass it the ID it calls the post object itself. If you pass it the post object, then you save the application from one more big object loading function.

**Pull request merged**